src/styles/button.scss

```diff
@@ -0,0 +1,24 @@
+.to-buy {
+  display: block;
+  width: 166px;
+  margin-left: $padding-left;
```

Comment:
The variable `$padding-left` is used but not defined. Please define this variable to avoid compilation errors.